### PR TITLE
bump version and fix release step

### PR DESCRIPTION
**Pull request by Korijn**
* Bump version to 0.3.4
* Copy publish step from pygfx repo

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -83,36 +83,30 @@ jobs:
...
         token: ${{ secrets.GITHUB_TOKEN }}
+        files: |
+          dist/*.tar.gz
+          dist/*.whl
     - name: Publish to PyPI
       uses: pypa/gh-action-pypi-publish@release/v1
       with:
         user: __token__
-        password: ${{ secrets.PYPI_PASSWORD }}
+        password: ${{ secrets.PYPI_PASSWORD }}
```
Comment (FirefoxMetzger) [inline .github/workflows/ci.yml:112]:
```suggestion
        password: ${{ secrets.PYPI_PASSWORD }}

```

Finish file with new line.

File: pylinalg/__init__.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 # flake8: noqa
 
-__version__ = "0.3.3"
+__version__ = "0.3.4"
```
Comment (FirefoxMetzger) [inline pylinalg/__init__.py:9]:
```suggestion
import pkg_resources

__version__ = pkg_resources.get_distribution('pylinalg').version
```

Shall we can reduce the places where we have to bump version numbers by 1?

**Review by FirefoxMetzger — APPROVED** [review]
Looks good 👍 Two small suggestions to make the repo a bit more tidy.

File: pylinalg/__init__.py (hunk 2 above)
Comment (Korijn) [reply pylinalg/__init__.py:9]: Sure, but then we should use importlib as it is replacing pkg_resources. Let's do that in a future PR.

**Review by FirefoxMetzger — APPROVED** [review]

**Pull request merged**